client/src/components/Tickets/TicketCreateForm.jsx

```diff
@@ -156,8 +162,8 @@ function TicketCreateForm({
                   id="attachments"
                   isSubmitted={isSubmitting}
                   description="Error reports and screenshots can be helpful for diagnostics"
-                  maxSizeMessage="Max File Size: 3MB"
-                  maxSize={3145728}
+                  maxSizeMessage={maxSizeMessage || 'Max File Size: 3MB'}
+                  maxSize={maxSize || 3145728}
```

Comment:
@jmartinez-tacc We missed a spot. The same change needs to be made in `client/src/components/Tickets/TicketModal.jsx`